### PR TITLE
Allow overriding aws-cli execution timeout via environment variable

### PR DESCRIPTION
**Pull request by nohuhu**

### CODE REVIEW

File: lib/AWS/CLIWrapper.pm [hunk 1]
```diff
@@ -194,7 +195,7 @@ sub _execute {
         # better for perl debugger
         my($ok, $err, $buf, $stdout_buf, $stderr_buf) = IPC::Cmd::run(
             command => join(' ', @cmd),
-            timeout => $opt{timeout} || 30,
+            timeout => $opt{timeout} || $self->{timeout},
```
Comment (nohuhu) [inline lib/AWS/CLIWrapper.pm:198]: Note that this code does not allow `$opt{timeout}` to be set to zero, I'm not sure how meaningful that timeout value would be so I didn't change it.

File: lib/AWS/CLIWrapper.pm [hunk 2]
```diff
@@ -208,7 +209,7 @@ sub _execute {
         print "";
     } else {
         $ret = IPC::Cmd::run_forked(join(' ', @cmd), {
-            timeout => $opt{timeout} || 30,
+            timeout => $opt{timeout} || $self->{timeout},
```
Comment (nohuhu) [inline lib/AWS/CLIWrapper.pm:212]: Ditto.

**Review by hirose31 — APPROVED** [review]
Thanks!!

**Comment by hirose31** [comment]
I've just released v1.24, it will be soon available.

**Pull request merged**